Autocomplete/basic/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+
+    <link rel="manifest" href="./manifest.webmanifest" />
+    <link rel="shortcut icon" href="./favicon.png" />
+
+    <link rel="stylesheet" href="src/style.css" />
+    <link rel="stylesheet" href="src/predefinedItemsPlugin.css" />
+    <link rel="stylesheet" href="src/hits.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/instantsearch.css@7/themes/satellite-min.css"
+    />
+
+    <title>autocomplete-results-page</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>Basic Standalone Autocomplete</h1>
```

Comment:
```suggestion
      <h1>Multi-source Autocomplete</h1>
```
I think "basic" and "standalone" might be confusing here. "Multi-source" is descriptive without being overkill (e.g., "Autocomplete with pre-defined items, recent searches, and suggested searches"). 

True "source" might be confusing, but a little more description below can explain that.

Reply:
Makes sense!